Source/Plugin.BLE.Abstractions/CharacteristicBase.cs

```diff
@@ -102,18 +102,18 @@ private CharacteristicWriteType GetWriteType()
                 CharacteristicWriteType.WithoutResponse;
         }
 
-        public Task StartUpdatesAsync()
+        public Task StartUpdatesAsync(CancellationToken cancellationToken = default)
         {
             if (!CanUpdate)
             {
                 throw new InvalidOperationException("Characteristic does not support update.");
             }
 
             Trace.Message("Characteristic.StartUpdates");
-            return StartUpdatesNativeAsync();
+            return StartUpdatesNativeAsync(cancellationToken);
         }
 
-        public Task StopUpdatesAsync()
+        public Task StopUpdatesAsync(CancellationToken cancellationToken = default)
```

Comment:
Generally LGTM, but here the token is not being passed on to `StopUpdatesNativeAsync`!

Reply:
Fixed in b917bf2e57c415d02b73f3f8f10ed07902d0a079.